js/script.js

```diff
@@ -0,0 +1,177 @@
+                                            <path class="checkmark-check" fill="none" d="M14.1 27.2l7.1 7.2 16.7-16.8"/>
+                                    </svg>
+                                    <strong>Sua mensagem foi enviada!</strong>
+                                    <div class="column is-narrow has-text-centered">
+                                        <button type="button" title="Retornar" class="button is-info back-button">Retornar</button>
+                                    </div>
+                                </div>`;
+        ajaxStatus = 'success';
+      }
+      if (status === 'fail') {
+        messageText = `<div class="error-box">
+                              <svg class="checkmark  checkmark-error" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 52 52">
+                                  <circle class="checkmark-circle" cx="26" cy="26" r="25" fill="none"/>
+                                  <path class="checkmark-check" fill="none" d="M16 16 36 36 M36 16 16 36"/>
+                              </svg>
+                              <strong>Ocorreu um erro :( <br/> Tente novamente mais tarde</strong>
+                              <div class="column is-narrow has-text-centered">
+                                  <button type="button" title="Retornar" class="button is-info back-button">Retornar</button>
+                              </div>
+                          </div>`;
+        ajaxStatus = 'fail';
+      }
+
+      const msgBox = `<div class="message-alert">
+                        <div class="is-${ajaxStatus}">
+                                ${messageText}
+                        </div>
+                     </div>`;
+
+      return msgBox;
+    },
+
+    updateMsg: (nodes, msg) => {
+      nodes.insertAdjacentHTML('beforeend', msg);
+    },
+
+    return: () => {
+      const { form } = startForm.ui;
+      const { fields } = startForm.ui;
+      const messageBox = document.querySelectorAll('.message-alert');
+
+      form.style.display = 'block';
+
+      fields.forEach((field) => {
+        field.value = '';
+      });
+
+      messageBox.forEach((box) => {
+        box.remove();
+      });
+    },
+
+    sendData: (e) => {
+      e.preventDefault();
+
+      const initFunctions = startForm.functions;
+      const container = document.querySelector('.container');
+
+      const { form } = startForm.ui;
+      const { sendButton } = startForm.ui;
+
+      sendButton.disabled = true;
+
+      const request = new XMLHttpRequest();
+      request.open('POST', '//formspree.io/matheusbaroneteste@gmail.com', true);
+      request.setRequestHeader('accept', 'application/json');
+
+      const formData = new FormData(form);
+      request.send(formData);
+
+      request.onreadystatechange = () => {
+        if (request.readyState === 3) {
+          form.style.display = 'none';
+          initFunctions.updateMsg(container, initFunctions.formMessage('loading'));
```

Comment:
you wrote initFunctions.updateMsg here and few lines bellow, why not destruct it with the rest of variables declarations?
```
const { updateMsg } = initFunctions
```